src/java.desktop/unix/classes/sun/awt/X11/XDecoratedPeer.java

```diff
@@ -1305,4 +1318,24 @@ public void handleWindowFocusOut(Window oppositeWindow, long serial) {
         }
         super.handleWindowFocusOut(oppositeWindow, serial);
     }
+
+    public static final String MWM_DECOR_TITLE_PROPERTY_NAME = "xawt.mwm_decor_title";
+
+    public final Optional<Boolean> getMWMDecorTitleProperty() {
+        Optional<Boolean> res = Optional.empty();
+
+        if (target instanceof javax.swing.RootPaneContainer) {
```

Comment:
Looks like this instanceof will load the Swing as well.

Reply:
Sorry, missed that one the first time. Corrected now.